Header header
uint32 blob_count
MappedBlob[] blobs